client/app/tests/phpunit/Service/File/FileNameFixerTest.php

```diff
@@ -83,4 +86,25 @@ public function unusualCharactersProvider()
             'Directory constructs are not allowed' => ['../../', '___.'],
         ];
     }
+
+    /**
+     * @test
+     *
+     * @dataProvider uploadedFileDataProvider
+     */
+    public function lowerCaseFileExtension($filePath, $uploadedFileNameAndExtension, $expectedFileNameAndExtension)
+    {
+        $uploadedFile = new UploadedFile(sprintf('%s'.$filePath, $this->projectDir), $uploadedFileNameAndExtension);
+        $uploadedFileExtensionCheck = $this->sut->lowerCaseFileExtension($uploadedFile);
+
+        self::assertEquals($expectedFileNameAndExtension, $uploadedFileExtensionCheck->getClientOriginalName());
+    }
+
+    public function uploadedFileDataProvider()
```

Comment:
You should probably add a test for the filename extension being blank, as there is a conditional statement for that case.

Reply:
Thanks for the reminder Elliot, I'd committed this but forget to push it up